schema/Mutation.graphql

```diff
@@ -84,4 +84,9 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create new life
+    """
+    createLife(firstName: String!, lastName: String!, description: String!, birthday: DateTime!, hobbies: [String!]!, title: String!): Life!
```

Comment:
You can create an `input` type object for this so it is cleaner to read

Reply:
Would I create the `input` in the `Mutation.graphql` file or in a separate file?